.backstage/backstage-go.yaml

```diff
@@ -7,6 +7,9 @@ metadata:
     - url: https://pkg.go.dev/go.einride.tech/backstage
       title: GoDoc
       icon: docs
+  annotations:
+    github.com/project-slug: einride/backstage-go
```

Comment:
What is the benefit of adding this again? IMO it feels redundant, I would rather not have to add this manually everywhere unless it really creates value

Reply:
Mostly to get as much data as possible into Backstage, but I'm fine with not adding it 😄